lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -316,6 +316,13 @@
   window.webkitStorageInfo.PERSISTENT // FAIL
 }
 
+function isOnHttps() {
+  // Check blob URL.
+  const img = document.createElement('img');
+  img.src = URL.createObjectURL(blob);
```

Comment:
@ebidel I don't think this is actually working? `Uncaught ReferenceError: blob is not defined`.

The expectation after this change should have `length: 2` (the current extendedInfo has `http://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js` in it, not the blob URL)